crates/pallet-bridge-pot-currency-swap/src/tests.rs

```diff
@@ -1,44 +1,135 @@
+use frame_support::sp_runtime::ArithmeticError;
+
 use crate::mock::*;
+use crate::{Balanced, BalancedError};
 
-/// This test verifies that the genesis builder correctly ensures genesis verifier result in case
-/// it returns false.
+/// This test verifies that the genesis builder correctly ensures genesis bridge pot balances
+/// values that are balanced.
 #[test]
-#[should_panic = "invalid genesis bridge pot currency swap related data"]
-fn genesis_verifier_false() {
+fn genesis_balanced_true() {
```

Comment:
Add another test like this but with only bridge pot accounts (i.e. no other accounts)

Reply:
Added